Public/Src/Engine/UnitTests/Scheduler.IntegrationTest/SchedulerIntegrationTestBase.cs

```diff
@@ -31,6 +31,7 @@
 using Xunit.Abstractions;
 using AssemblyHelper = BuildXL.Utilities.AssemblyHelper;
 using ProcessOutputs = BuildXL.Pips.Builders.ProcessOutputs;
+using System.Threading;

```

Comment:
>stem.Threadi [](start = 8, length = 12)

nit: Didn't see used and System.* should be at the top of the usings according to our .formatting style file

Reply:
fixed